### ISSUE TITLE
Use same versioning for tags and bower.json

### ISSUE THREAD

**Issue by alex88** [jquery-form/form#455]
Versions doesn't match, tags are for example `3.51` and bower.json shows `3.51.0`, that makes services like https://rails-assets.org/ never pick up the version since its mismatch.

**Comment by vjrabanelly** [comment]
:+1: Even bower is complaining about it 

``` shell
$ bower list
├─┬ jquery-form#3.51 incompatible with 3.51 (latest is 3.46.0)
│ └── jquery#2.1.3
```

**Comment by colemanw** [comment]
+1 Bower reports `3.46.0` as the latest version

**Comment by kevindb** [comment]
Probably resolved by 0dc9b4e
The new release has just been posted. I'll check back in a day or two to see if the package managers have picked up on the new version.

**Comment by kevindb** [comment]
Bower has been updated with [v4.1.0](https://github.com/jquery-form/form/releases/tag/v4.1.0)
